packages/mdc-tab-scroller/mdc-tab-scroller.scss

```diff
@@ -0,0 +1,50 @@
+/**
+  * @license
+  * Copyright 2018 Google Inc. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License")
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+@import "@material/animation/variables";
+
+.mdc-tab-scroller {
+  overflow-y: hidden;
+}
+
+.mdc-tab-scroller__scroll-area {
+  -webkit-overflow-scrolling: touch;
+  display: flex;
+  margin-bottom: -18px;
```

Comment:
why 18px?

Reply:
It's the size of most scrollbars. I'll make this a variable and add a comment explaining it.

Reply:
Following up in #3059